pipelines/batch/src/main/java/com/google/fhir/analytics/FhirEtlOptions.java

```diff
@@ -265,4 +265,18 @@ public interface FhirEtlOptions extends BasePipelineOptions {
   String getSourceNdjsonFilePatternList();
 
   void setSourceNdjsonFilePatternList(String value);
+
+  @Description(
+      "Whether to use HAPI-FHIR's MDM service to replace Resource references with their Golden"
+          + " Resource counterparts")
+  @Default.Boolean(false)
+  Boolean getMapToGoldenResources();
+
+  void setMapToGoldenResources(Boolean mapToGoldenResources);
+
+  @Description("Comma separated list of resources to treat as MDM Resources.")
+  @Default.String("Patient")
```

Comment:
**todo:** I don't think this should have a default value. From the point of view of getting this merged into the mainline, not all users of `fhir-data-pipes` will be using HAPI-FHIR's MDM.